---
title: "Curriculum Vitae"
---

### Education

## PhD

###### **Deakin University:** September 2016 to present

###### Currently pursuing a Doctor of Philosophy with a concentration in bioinformatics and medical genomics

## MD

###### **SUNY Upstate Medical University:** August 2011 to May 2016

###### Graduated with a Medicinae Doctor, devoting credited hours to research in bioinformatics and medical genomics

## BS

###### **SUNY New Paltz:** August 2009 to May 2011

###### Graduated *summa cum laude*, obtaining a Bachelor’s Degree in Biology with a concentration in biotechnology

## AS

###### **SUNY Orange:** January 2007 to May 2009

###### Graduated *summa cum laude*, obtaining an Associate’s Degree in Science with a concentration in biology

<br>

### Open source software

#### R packages which I designed and maintain,

##### **exprso** | [CRAN](https://CRAN.R-project.org/package=exprso)

###### This package introduces a framework for the rapid implementation of high-throughput supervised machine learning built with the biologist user in mind. Written by biologists, for biologists, this package provides a user-friendly interface that empowers investigators to execute state-of-the-art binary and multi-class classification, including deep learning, with minimal programming experience necessary.

##### **peakRAM** | [CRAN](https://CRAN.R-project.org/package=peakRAM)

###### When working with big data sets, RAM conservation is critically important. However, it is not always enough to just monitor the size of the objects created. So-called "copy-on-modify" behavior, characteristic of R, means that some expressions or functions may require an unexpectedly large amount of RAM overhead. For example, replacing a single value in a matrix duplicates that matrix in the back-end, making this task require twice as much RAM as that used by the matrix itself. This package makes it easy to monitor the total and peak RAM used so that developers can quickly identify and eliminate RAM hungry code.

##### **propr** | [CRAN](https://CRAN.R-project.org/package=propr)

###### The bioinformatic evaluation of gene co-expression often begins with correlation-based analyses. However, this approach lacks statistical validity when applied to relative data. This includes, for example, biological count data produced by high-throughput RNA-sequencing, chromatin immunoprecipitation (ChIP), ChIP-sequencing, Methyl-Capture sequencing, and other techniques. This package implements several measures of dependence between relative features using compositional data analysis.

#### R packages to which I contributed,

##### **ALDEx2** | [Bioconductor](https://www.bioconductor.org/packages/devel/bioc/html/ALDEx2.html)

###### This package performs a differential abundance analysis of biological count data using log-ratio transformations.

##### **catmap** | [CRAN](https://CRAN.R-project.org/package=catmap)

###### This package conducts a meta-analysis on combined case-control and family-based (TDT) genetic data.

<br>

### Professional experience

#### In industry,

##### **Adveq Management:** April 2016 to present

###### Worked to develop an object-oriented back-end in R for organizing, modeling, and analyzing transactions between financial institutions. Position held as a short-term, free-lance contract.

#### In research,

##### **SUNY Upstate Medical University:** August 2012 to May 2016

###### Worked in a psychiatric genetics lab under Dr. Stephen Glatt to study the molecular underpinnings of psychiatric disorders. Efforts largely included the development of bioinformatic tools for the analysis of single nucleotide polymorphisms and microarray-based gene expression data. Extensive time devoted to the implementation machine learning for the classification of autism spectrum disorder using genomic data.

##### **SUNY New Paltz:** September 2009 to August 2010

###### Worked in an organic chemistry lab under Dr. Frantz Folmer-Andersen to study synthetic diamine macrocycles which have the capability to distinguish between mirror image molecules.

##### **SUNY Purchase:** May 2009 to July 2009

###### Worked in an environmental science lab under oceanographer Dr. Kristopher Baker to study the impact of anthropogenic input on the biodiversity of the Hudson River estuary. Investigation presented as lecture.

#### In education,

##### **Deakin University:** March 2017 to present

###### Employed as a graduate assistant for two courses titled, *Principles of Pharmacology* and *Therapeutic Development*.

##### **Coursera:** September 2016 to October 2016

###### Employed as a graduate assistant for an online course titled, *Big Data, Genes, and Medicine*.

##### **SUNY New Paltz:** September 2009 to May 2011

###### Employed as a teaching assistant for an undergraduate organic chemistry laboratory class. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.

##### **SUNY Orange:** September 2008 to August 2009

###### Employed in a math learning center to assist students in need of help in college mathematics courses. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.

<br>

### Awards and scholarships

2017 Travel Award to Winter School in Mathematical & Computational Biology

2016 Travel Award to 2nd Bioconductor Asia-Pacific Developers' Meeting

2014 Student Research Celebration Presenter Research Project Award

2010 New Paltz Student Undergraduate Research Experience (SURE) stipend

2009 Purchase Bridges to Baccalaureate Program stipend

2009 Outgoing Faculty Annual Scholarship awarded by SUNY Orange

2009 Outgoing William F. Ehlers Jr. Memorial Scholarship for excellence in anthropology awarded by SUNY Orange

2009 Outgoing Dr. Cortland R. Mapes Scholarship for excellence in biology awarded by SUNY Orange

2008 American Chemical Society award for excellence in chemistry

2007-2009 Recipient of President’s Scholarship for Outstanding Academic Achievement at SUNY Orange

<br>

### Publications

Erb I, **Quinn T**, Lovell D, & Notredame C. 2017. Differential Proportionality - A Normalization-Free Approach To Differential Gene Expression. bioRxiv 134536. http://dx.doi.org/10.1101/134536.

Tylee DS, Kikinis Z, **Quinn TP**, et al., & Makris N. 2017. Machine-learning classification of 22q11.2 deletion syndrome: A diffusion tensor imaging study. NeuroImage: Clinical 15(1):832-42. http://dx.doi.org/10.1016/j.nicl.2017.04.029.

**Quinn T**, Richardson MF, Lovell D, & Crowley T. 2017. propr: An R-package for Identifying Proportionally Abundant Features Using Compositional Data Analysis. bioRxiv 104935. http://dx.doi.org/10.1101/104935.

Tylee DS, Hess JL, **Quinn TP**, et al., & Glatt SJ. 2016. Blood transcriptomic comparison of individuals with and without autism spectrum disorder: A combined-samples mega-analysis. American Journal of Medical Genetics B Neuropsychiatric Genetics. Rapid Publication. http://dx.doi.org/10.1002/ajmg.b.32511.

**Quinn T**, Tylee D, & Glatt S. 2016. exprso: an R-package for the rapid implementation of machine learning algorithms. F1000Research 5:2588. http://dx.doi.org/10.12688/f1000research.9893.1.

Bichindaritz IB, Cole E, Keshan N, Parimi P, & **Quinn TP**. 2016. Machine Learning Based Automatic Multilevel Stress Detection from ECG Signals. IJCAI 2016: Workshop on Knowledge Discovery in Healthcare Data.

Hess JL, **Quinn TP**, Akbarian S, & Glatt SJ. 2015. Bioinformatic analyses and conceptual synthesis of evidence linking ZNF804A to risk for schizophrenia and bipolar disorder. American Journal of Medical Genetics B Neuropsychiatric Genetics 168(1):14-35. http://dx.doi.org/10.1002/ajmg.b.32284.

**Quinn TP**, Atwood PD, Tanski JM, Moore TF, & Folmer-Andersen J. 2011. Aza-crown macrocycles as chiral solvating agents for mandelic acid derivatives. Journal of Organic Chemistry 76(24):10020-30. http://dx.doi.org/10.1021/jo2018203.

<br>
