---
title: "Curriculum Vitae"
output: pdf_document
---

### Education

## PhD

###### **Deakin University:** September 2016 to July 2020

###### Graduated with a Doctor of Philosophy, with a thesis titled "Understanding sequencing data as compositions"

## MD

###### **SUNY Upstate Medical University:** August 2011 to May 2016

###### Graduated with a Medicinae Doctor, with credited research hours in bioinformatics and medical genomics

## BS

###### **SUNY New Paltz:** August 2009 to May 2011

###### Graduated *summa cum laude*, obtaining a Bachelor’s Degree in Biology with a concentration in biotechnology

## AS

###### **SUNY Orange:** January 2007 to May 2009

###### Graduated *summa cum laude*, obtaining an Associate’s Degree in Science with a concentration in biology

<br>

### Open source software

#### R packages which I designed and maintain,

##### **balance** | [CRAN](https://cran.r-project.org/package=balance)

###### Balances have become a cornerstone of compositional data analysis. However, conceptualizing balances is difficult, especially for high-dimensional data. Most often, investigators visualize balances with "balance dendrograms". However, this visualization tool does not scale well for large data. This package provides an alternative scheme for visualizing balances, described in [Quinn (2018)]. This package also provides a method for principal balance analysis.

##### **exprso** | [CRAN](https://CRAN.R-project.org/package=exprso)

###### Supervised machine learning has an increasingly important role in data analysis. This package introduces a framework for rapidly building and deploying supervised machine learning in a high-throughput manner. This package provides a user-friendly interface that empowers investigators to execute state-of-the-art binary and multi-class classification, as well as regression, with minimal programming experience necessary.

##### **peakRAM** | [CRAN](https://CRAN.R-project.org/package=peakRAM)

###### When working with big data sets, RAM conservation is critically important. However, it is not always enough to just monitor the size of the objects created. So-called "copy-on-modify" behavior, characteristic of R, means that some expressions or functions may require an unexpectedly large amount of RAM overhead. For example, replacing a single value in a matrix duplicates that matrix in the back-end, making this task require twice as much RAM as that used by the matrix itself. This package makes it easy to monitor the total and peak RAM used so that developers can quickly identify and eliminate RAM hungry code.

##### **propr** | [CRAN](https://CRAN.R-project.org/package=propr)

###### The bioinformatic evaluation of gene co-expression often begins with correlation-based analyses. However, correlation lacks validity when applied to relative data, including count data generated by next-generation sequencing. This package implements several metrics for proportionality, including phi [Lovell et al (2015)] and rho [Erb and Notredame (2016)]. This package also implements several metrics for differential proportionality. Unlike correlation, these measures give the same result for both relative and absolute data.

#### R packages to which I contributed,

##### **ALDEx2** | [Bioconductor](https://www.bioconductor.org/packages/devel/bioc/html/ALDEx2.html)

###### This package performs a differential abundance analysis of biological count data using log-ratio transformations.

##### **catmap** | [CRAN](https://CRAN.R-project.org/package=catmap)

###### This package conducts a meta-analysis on combined case-control and family-based (TDT) genetic data.

<br>

### Professional experience

#### In research,

##### **Deakin University:** October 2019 to present

###### Employed as an **Associate Research Fellow** in the Applied Artificial Intelligence Institute (A2I2) lab under Dr. Svetha Venkatesh to study XAI for health applications. This role involved close collaboration with computer science experts to design and implement neural network architectures that enable the interpretable analysis of high-dimensional genomic data.

##### **Deakin University:** January 2018 to August 2019

###### Employed as a **Casual Research Fellow** in the Pattern Recognition and Data Analysis (PRaDA) lab under Dr. Svetha Venkatesh to study autism spectrum disorder and cancer genomics. This role involved pre-processing genomic data, building and deploying classifiers based on gene expression signatures, and supervising a research assistant.

##### **SUNY Upstate Medical University:** August 2012 to May 2016

###### Employed as a **Student Research Assistant** in the Psychiatric Genetic Epidemiology and Neurobiology (PsychGENe) lab under Dr. Stephen Glatt to study the molecular underpinnings of psychiatric disorders. This role involved the development of bioinformatic tools for the analysis of single nucleotide polymorphisms and microarray-based gene expression data. Extensive time devoted to the implementation machine learning for the classification of autism spectrum disorder.

##### **SUNY New Paltz:** September 2009 to August 2010

###### Employed as a **Student Research Assistant** in an organic chemistry lab under Dr. Frantz Folmer-Andersen to study synthetic diamine macrocycles which have the capability to distinguish between mirror image molecules.

##### **SUNY Purchase:** May 2009 to July 2009

###### Employed as a **Student Research Assistant** in an environmental science lab under Dr. Kristopher Baker to study the impact of anthropogenic input on the biodiversity of the Hudson River estuary.

#### In education,

##### **Deakin University:** March 2017 to May 2019

###### Employed as a **Graduate Assistant** for two courses titled, *Principles of Pharmacology* (x3) and *Therapeutic Development* (x2).

##### **Coursera:** September 2016 to October 2016

###### Employed as a **Graduate Assistant** for an online course titled, *Big Data, Genes, and Medicine*.

##### **SUNY New Paltz:** September 2009 to May 2011

###### Employed as a **Teaching Assistant** for an undergraduate organic chemistry laboratory class.

##### **SUNY Orange:** September 2008 to August 2009

###### Employed as a **Tutor** in a university math learning center.

#### In industry,

##### **Adveq Management:** April 2016 to August 2017

###### Employed as a **Data Scientist** to develop an object-oriented back-end in R for organizing, modeling, and analyzing transactions between financial institutions. Position held as a short-term, free-lance contract.

<br>

### Community engagement

##### **ABACBS:** October 2020

###### Reviewed abstracts as part of the ABACBS 2020 Program Committee.

##### **NAR Genomics and Bioinformatics:** December 2019 to December 2020

###### Served as guest editor for special issue on compositional data analysis.

##### **COMBINE:** October 2018

###### Reviewed abstracts as part of the COMBINE 2018 Program Committee.

##### **Geelong Hacky Hour:** October 2017 to October 2019

###### Co-founder of the Geelong Hacky Hour chapter where researchers help other researchers learn to program.

<br>

### Awards and scholarships

2020 Shortlisted for Alfred Deakin Medal for Doctoral Thesis

2019 People's Choice Award for Outstanding Talk at Joint GIW/ABACBS 2019

2017 Travel Award to Winter School in Mathematical & Computational Biology

2016 Travel Award to 2nd Bioconductor Asia-Pacific Developers' Meeting

2014 Student Research Celebration Presenter Research Project Award

2010 New Paltz Student Undergraduate Research Experience (SURE) stipend

2009 Purchase Bridges to Baccalaureate Program stipend

2009 Outgoing Faculty Annual Scholarship awarded by SUNY Orange

2009 Outgoing William F. Ehlers Jr. Memorial Scholarship for excellence in anthropology awarded by SUNY Orange

2009 Outgoing Dr. Cortland R. Mapes Scholarship for excellence in biology awarded by SUNY Orange

2008 American Chemical Society award for excellence in chemistry

2007-2009 Recipient of President’s Scholarship for Outstanding Academic Achievement at SUNY Orange

<br>

### Presentations

**Quinn TP**. December 2020. Deep learning: The quest for interpretability. Invited talk presented at Centre for Genomic Regulation in Barcelona, Spain (online delivery).

**Quinn TP**. November 2020. Deep learning: The quest for interpretability. Invited talk presented at WEHI ML Interest Group Seminar Series in Melbourne, Australia (online delivery).

**Quinn TP**. November 2020. HyperXPair: Learning distance-dependent motif interactions. Accepted talk presented at ABACBS 2020 Virtual Conference (online delivery).

**Quinn TP**. September 2020. Risk Prediction: An Introduction. Invited talk presented at Victoria University Bioinformatics Seminar in Melbourne, Australia (online delivery).

**Quinn TP**. July 2020. DeepCoDA: Personalized Interpretability for Compositional Health Data. Accepted talk presented at ICML 2020 in Vienna, Austria (online delivery).

**Quinn TP**. July 2020. On Compositions and Deep Learning. Invited talk presented at Monash Bioinformatics Seminar Series in Melbourne, Australia (online delivery).

**Quinn TP**. June 2020. DeepCoDA: Personalized Interpretability for Compositional Health Data. Invited talk presented at Gen(e)quality "Demystifying 'omic' Risk Scores" workshop in Geelong, Australia (online delivery).

**Quinn TP**. June 2020. Risk Prediction: An Introduction. Invited talk presented at Gen(e)quality "Demystifying 'omic' Risk Scores" workshop in Geelong, Australia (online delivery).

**Quinn TP**. May 2020. The microbiome as compositions: a methodological survey. Invited talk presented at San Diego State University in San Diego, California (online delivery).

**Quinn TP**. December 2019. Deep in the Bowel: Highly Interpretable Neural Encoder-Decoder Networks Predict Gut Metabolites from Gut Microbiome. Accepted talk presented at Joint GIW/ABACBS 2019 in Sydney, Australia.

**Quinn TP**. December 2019. DeepTRIAGE: Interpretable and Individualised Biomarker Scores using Attention Mechanism for the Classification of Breast Cancer Sub-types. Accepted talk presented at Joint GIW/ABACBS 2019 in Sydney, Australia.

**Quinn TP**. June 2019. Using balances to engineer features for the classification of health biomarkers: a new approach to balance selection. Accepted talk presented at CoDaWork 2019 in Terrassa, Spain.

**Quinn TP**. March 2019. A practical guide to (supervised) machine learning using the exprso R package. Invited talk presented at Deakin University in Geelong, Victoria.

**Quinn TP**. October 2018. Understanding sequencing data as compositions: theory and application. Invited talk presented at Centre for Genomic Regulation in Barcelona, Spain.

**Quinn TP**. October 2017. A compositionally valid pipeline for any-omics data. Accepted talk presented at Australasian Genomic Technologies Association in Hobart, Tasmania.

<br>

### Peer-review contributions

[Publons](https://publons.com/researcher/1557877/thom-quinn/)

Wellcome Trust

Statistical Methods in Medical Research

PeerJ

American Journal of Medical Genetics B Neuropsychiatric Genetics

Bioinformatics

Applied Computing and Geosciences

NAR Genomics and Bioinformatics

PLoS Computational Biology

<br>

### Lead Author Publications

**Quinn TP**, Jacobs S, Senadeera M, Le V, & Coghlan S. 2020. The Three Ghosts of Medical AI: Can the Black-Box Present Deliver? arXiv. http://arxiv.org/abs/2012.06000.

**Quinn TP**, Senadeera M, Jacobs S, Coghlan S, & Le V. 2020. Trust and Medical AI: The challenges we face and the expertise needed to overcome them. JAMIA. Online ahead of print. http://doi.org/10.1093/jamia/ocaa268.

**Quinn TP** & Erb I. 2020. Amalgams: data-driven amalgamation for the dimensionality reduction of compositional data. NAR Genomics and Bioinformatics 2(4). http://doi.org/10.1093/nargab/lqaa076.

**Quinn TP**, Le V, & Cardilini APA. 2020. Test Set Verification Is An Essential Step in Model Building. Methods in Ecology and Evolution. Forum Article. http://doi.org/10.1111/2041-210X.13495.

Le V+, **Quinn TP+**, Tran T, & Venkatesh S. 2020. Deep in the Bowel: Highly Interpretable Neural Encoder-Decoder Networks Predict Gut Metabolites from Gut Microbiome. BMC Genomics 21(256). http://doi.org/10.1186/s12864-020-6652-7.

**Quinn TP**, Nguyen D, Nguyen P, Gupta S, & Venkatesh S. 2020. Learning distance-dependent motif interactions: an interpretable CNN model of genomic events. bioRxiv 270967. http://doi.org/10.1101/2020.08.27.270967.

**Quinn TP+**, Nguyen D+, Rana S, Gupta S, & Venkatesh S. 2020. DeepCoDA: personalized interpretability for compositional health data. ICML 2020 (A* conference, acceptance rate = 21.8%). http://arxiv.org/abs/2006.01392.

Loughman A+, **Quinn TP+**, et al., & Tang ML. 2020. Infant microbiota in colic: predictive associations with problem crying and subsequent child behavior. Journal of Developmental Origins of Health and Disease. Online ahead of print. http://doi.org/10.1017/S2040174420000227.

**Quinn TP** & Erb I. 2020. Interpretable Log Contrasts for the Classification of Health Biomarkers: a New Approach to Balance Selection. mSystems 5(2). http://doi.org/10.1128/mSystems.00230-19.

Beykikhoshk A+, **Quinn TP+**, Lee SC, Tran T, & Venkatesh S. 2020. DeepTRIAGE: Interpretable and Individualised Biomarker Scores using Attention Mechanism for the Classification of Breast Cancer Sub-types. BMC Medical Genomics 13(20). http://doi.org/10.1186/s12920-020-0658-5.

**Quinn TP** & Erb I. 2019. Another look at microbe–metabolite interactions: how scale invariant correlations can outperform a neural network. bioRxiv 847475. http://doi.org/10.1101/847475.

Harikumar H+, **Quinn TP+**, Rana S, Gupta S, & Venkatesh S. 2019. A random walk down personalized single-cell networks: predicting the response of any gene to any drug for any patient. bioRxiv 837807. http://doi.org/10.1101/837807.

**Quinn TP**, Erb I, Gloor G, Notredame C, Richardson MF, & Crowley TM. 2019. A field guide for the compositional analysis of any-omics data. GigaScience giz107. http://doi.org/10.1093/gigascience/giz107.

Lee SC+, Quinn A, Nguyen T, Venkatesh S, & **Quinn TP+**. 2019. A cross-cancer metastasis signature in the microRNA-mRNA axis of paired tissue samples. Molecular Biology Reports. http://doi.org/10.1007/s11033-019-05025-w.

**Quinn TP**, Nguyen T, Lee SC, & Venkatesh S. 2019. Cancer as a tissue anomaly: classifying tumor transcriptomes based only on healthy data. Frontiers in Genetics 10:599. http://doi.org/10.3389/fgene.2019.00599.

**Quinn TP+**, Lee SC+, Venkatesh S, & Nguyen T. 2019. Improving the classification of neuropsychiatric conditions using gene ontology terms as features. American Journal of Medical Genetics B Neuropsychiatric Genetics 180(7):508-18. http://doi.org/10.1002/ajmg.b.32727.

Lee SC+, **Quinn TP+**, et al., & Nguyen T. 2018. Solving for X: evidence for sex-specific autism biomarkers across multiple transcriptomic studies. American Journal of Medical Genetics B Neuropsychiatric Genetics 180(6):377-89. http://doi.org/10.1002/ajmg.b.32701.

**Quinn TP**. 2018. Visualizing Balances of Compositional Data: A New Alternative to Balance Dendrograms [version 1; referees: 2 approved]. F1000Research 7:1278. http://doi.org/10.12688/f1000research.15858.1.

**Quinn TP**, Crowley TM, & Richardson MF. 2018. Benchmarking differential expression analysis tools for RNA-Seq: normalization-based vs. log-ratio transformation-based methods. BMC Bioinformatics 19(1). http://doi.org/10.1186/s12859-018-2261-8.

**Quinn TP**, Erb I, Richardson MF, & Crowley TM. 2018. Understanding sequencing data as compositions: an outlook and review. Bioinformatics 34(16). http://doi.org/10.1093/bioinformatics/bty175.

**Quinn T**, Richardson MF, Lovell D, & Crowley T. 2017. propr: An R-package for Identifying Proportionally Abundant Features Using Compositional Data Analysis. Scientific Reports 7:16252. http://dx.doi.org/10.1038/s41598-017-16520-0.

**Quinn T**, Tylee D, & Glatt S. 2017. exprso: an R-package for the rapid implementation of machine learning algorithms [version 2; referees: 2 approved]. F1000Research 5:2588. http://dx.doi.org/10.12688/f1000research.9893.2.

**Quinn TP**, Atwood PD, Tanski JM, Moore TF, & Folmer-Andersen J. 2011. Aza-crown macrocycles as chiral solvating agents for mandelic acid derivatives. Journal of Organic Chemistry 76(24):10020-30. http://dx.doi.org/10.1021/jo2018203.

<br>

### Other Publications

Erb I, Gloor GB, & **Quinn TP**. 2020. Editorial: Compositional data analysis and related methods applied to genomics—a first special issue from NAR Genomics and Bioinformatics. NAR Genomics and Bioinformatics 2(4). http://doi.org/10.1093/nargab/lqaa103.

Nguyen T, Le H, **Quinn TP**, Le TD, & Venkatesh S. 2020. GraphDTA: Predicting drug–target binding affinity with graph neural networks. Bioinformatics. Online ahead of print. http://doi.org/10.1093/bioinformatics/btaa921.

Hess JL, Chen S, **Quinn TP**, et al., & Glatt SJ. 2020. BrainGENIE: The Brain Gene Expression and Network Imputation Engine. bioRxiv 356766. http://doi.org/10.1101/2020.10.27.356766.

Bruxel EM, Moreira-Maia CR, Akutagava-Martins GC, **Quinn TP**, et al., & Hutz MH. 2020. Meta-analysis and systematic review of ADGRL3 (LPHN3) polymorphisms in ADHD susceptibility. Molecular Psychiatry (2020). http://doi.org/10.1038/s41380-020-0673-0.

Nguyen T, Lee SC, **Quinn TP**, et al., & Le TD. 2019. Personalized Annotation-based Networks (PAN) for the Prediction of Breast Cancer Relapse. bioRxiv 534628. http://doi.org/10.1101/534628.

Allnutt TR, Wade B, **Quinn TP**, Richardson MF, & Crowley TM. 2018. Shortlisting Aptamer Candidates from HT-SELEX data. Aptamers 2. http://japtamers.co.uk/shortlisting-aptamer-candidates-from-ht-selex-data.

Erb I, **Quinn T**, Lovell D, & Notredame C. 2017. Differential Proportionality - A Normalization-Free Approach To Differential Gene Expression. Proceedings of CoDaWork 2017, The 7th Compositional Data Analysis Workshop; available under bioRxiv 134536. http://dx.doi.org/10.1101/134536.

Tylee DS, Kikinis Z, **Quinn TP**, et al., & Makris N. 2017. Machine-learning classification of 22q11.2 deletion syndrome: A diffusion tensor imaging study. NeuroImage: Clinical 15(1):832-42. http://dx.doi.org/10.1016/j.nicl.2017.04.029.

Englebert C, **Quinn T**, & Bichindaritz I. 2017. Feature selection for survival analysis in bioinformatics. CEUR Workshop Proceedings 1942:30-35. http://ceur-ws.org/Vol-1942/paper5.pdf.

Tylee DS, Hess JL, **Quinn TP**, et al., & Glatt SJ. 2016. Blood transcriptomic comparison of individuals with and without autism spectrum disorder: A combined-samples mega-analysis. American Journal of Medical Genetics B Neuropsychiatric Genetics 174(3):181-201. http://dx.doi.org/10.1002/ajmg.b.32511.

Bichindaritz IB, Cole E, et al., & **Quinn TP**. 2016. Machine Learning Based Automatic Multilevel Stress Detection from ECG Signals. IJCAI 2016: Workshop on Knowledge Discovery in Healthcare Data. http://sites.google.com/site/ijcai2016kdhealth/accepted-papers.

Hess JL, **Quinn TP**, Akbarian S, & Glatt SJ. 2015. Bioinformatic analyses and conceptual synthesis of evidence linking ZNF804A to risk for schizophrenia and bipolar disorder. American Journal of Medical Genetics B Neuropsychiatric Genetics 168(1):14-35. http://dx.doi.org/10.1002/ajmg.b.32284.

<br>

### Funding

Project: "Optimising treatments in mental health using AI". Year: 2020. Provisioner: MRFF Applied Artificial Intelligence Research in Health Fund. Award: $4,995,434. Role: CI. Contribution: Design and implementation of trial analysis protocol.

Project: "How does the gut microbiome contribute to individual responses to psychobiotics?". Year: 2020. Provisioner: Deakin IMPACT Seed Fund. Award: $15,000. Role: AI. Contribution: Design and implementation of statistical and machine learning analyses.

<br>
