<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Curriculum Vitae</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<link rel="icon" type="image/x-icon" href="favicon.ico" />




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">THOM QUINN</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about-me.html">ABOUT ME</a>
</li>
<li>
  <a href="curriculum-vitae.html">CURRICULUM VITAE</a>
</li>
<li>
  <a href="musings.html">MUSINGS</a>
</li>
<li>
  <a href="fiction.html">FICTION</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Curriculum Vitae</h1>

</div>


<div id="education" class="section level3">
<h3>Education</h3>
</div>
<div id="phd" class="section level2">
<h2>PhD</h2>
<div id="deakin-university-september-2016-to-present" class="section level6">
<h6><strong>Deakin University:</strong> September 2016 to present</h6>
</div>
<div id="currently-pursuing-a-doctor-of-philosophy-with-a-concentration-in-bioinformatics-and-medical-genomics" class="section level6">
<h6>Currently pursuing a Doctor of Philosophy with a concentration in bioinformatics and medical genomics</h6>
</div>
</div>
<div id="md" class="section level2">
<h2>MD</h2>
<div id="suny-upstate-medical-university-august-2011-to-may-2016" class="section level6">
<h6><strong>SUNY Upstate Medical University:</strong> August 2011 to May 2016</h6>
</div>
<div id="graduated-with-a-medicinae-doctor-devoting-credited-hours-to-research-in-bioinformatics-and-medical-genomics" class="section level6">
<h6>Graduated with a Medicinae Doctor, devoting credited hours to research in bioinformatics and medical genomics</h6>
</div>
</div>
<div id="bs" class="section level2">
<h2>BS</h2>
<div id="suny-new-paltz-august-2009-to-may-2011" class="section level6">
<h6><strong>SUNY New Paltz:</strong> August 2009 to May 2011</h6>
</div>
<div id="graduated-summa-cum-laude-obtaining-a-bachelors-degree-in-biology-with-a-concentration-in-biotechnology" class="section level6">
<h6>Graduated <em>summa cum laude</em>, obtaining a Bachelor’s Degree in Biology with a concentration in biotechnology</h6>
</div>
</div>
<div id="as" class="section level2">
<h2>AS</h2>
<div id="suny-orange-january-2007-to-may-2009" class="section level6">
<h6><strong>SUNY Orange:</strong> January 2007 to May 2009</h6>
</div>
<div id="graduated-summa-cum-laude-obtaining-an-associates-degree-in-science-with-a-concentration-in-biology" class="section level6">
<h6>Graduated <em>summa cum laude</em>, obtaining an Associate’s Degree in Science with a concentration in biology</h6>
<p><br></p>
</div>
<div id="open-source-software" class="section level3">
<h3>Open source software</h3>
<div id="r-packages-authored-and-maintained" class="section level4">
<h4>R packages authored and maintained,</h4>
<div id="exprso-cran" class="section level5">
<h5><strong>exprso</strong> | <a href="https://CRAN.R-project.org/package=exprso">CRAN</a></h5>
<div id="this-package-introduces-a-framework-for-the-rapid-implementation-of-high-throughput-supervised-machine-learning-built-with-the-biologist-user-in-mind.-written-by-biologists-for-biologists-this-package-provides-a-user-friendly-interface-that-empowers-investigators-to-execute-state-of-the-art-binary-and-multi-class-classification-including-deep-learning-with-minimal-programming-experience-necessary." class="section level6">
<h6>This package introduces a framework for the rapid implementation of high-throughput supervised machine learning built with the biologist user in mind. Written by biologists, for biologists, this package provides a user-friendly interface that empowers investigators to execute state-of-the-art binary and multi-class classification, including deep learning, with minimal programming experience necessary.</h6>
</div>
</div>
<div id="peakram-cran" class="section level5">
<h5><strong>peakRAM</strong> | <a href="https://CRAN.R-project.org/package=peakRAM">CRAN</a></h5>
<div id="when-working-with-big-data-sets-ram-conservation-is-critically-important.-however-it-is-not-always-enough-to-just-monitor-the-size-of-the-objects-created.-so-called-copy-on-modify-behavior-characteristic-of-r-means-that-some-expressions-or-functions-may-require-an-unexpectedly-large-amount-of-ram-overhead.-for-example-replacing-a-single-value-in-a-matrix-duplicates-that-matrix-in-the-back-end-making-this-task-require-twice-as-much-ram-as-that-used-by-the-matrix-itself.-this-package-makes-it-easy-to-monitor-the-total-and-peak-ram-used-so-that-developers-can-quickly-identify-and-eliminate-ram-hungry-code." class="section level6">
<h6>When working with big data sets, RAM conservation is critically important. However, it is not always enough to just monitor the size of the objects created. So-called “copy-on-modify” behavior, characteristic of R, means that some expressions or functions may require an unexpectedly large amount of RAM overhead. For example, replacing a single value in a matrix duplicates that matrix in the back-end, making this task require twice as much RAM as that used by the matrix itself. This package makes it easy to monitor the total and peak RAM used so that developers can quickly identify and eliminate RAM hungry code.</h6>
</div>
</div>
<div id="propr-cran" class="section level5">
<h5><strong>propr</strong> | <a href="https://CRAN.R-project.org/package=propr">CRAN</a></h5>
<div id="the-bioinformatic-evaluation-of-gene-co-expression-often-begins-with-correlation-based-analyses.-however-this-approach-lacks-statistical-validity-when-applied-to-relative-data.-this-includes-for-example-biological-count-data-produced-by-high-throughput-rna-sequencing-chromatin-immunoprecipitation-chip-chip-sequencing-methyl-capture-sequencing-and-other-techniques.-this-package-implements-several-measures-of-dependence-between-relative-features-using-compositional-data-analysis." class="section level6">
<h6>The bioinformatic evaluation of gene co-expression often begins with correlation-based analyses. However, this approach lacks statistical validity when applied to relative data. This includes, for example, biological count data produced by high-throughput RNA-sequencing, chromatin immunoprecipitation (ChIP), ChIP-sequencing, Methyl-Capture sequencing, and other techniques. This package implements several measures of dependence between relative features using compositional data analysis.</h6>
<p><br></p>
</div>
</div>
</div>
</div>
<div id="professional-experience" class="section level3">
<h3>Professional experience</h3>
<div id="in-industry" class="section level4">
<h4>In industry,</h4>
<div id="adveq-management-april-2016-to-present" class="section level5">
<h5><strong>Adveq Management:</strong> April 2016 to present</h5>
<div id="worked-to-develop-an-object-oriented-back-end-in-r-for-organizing-modeling-and-analyzing-transactions-between-financial-institutions.-position-held-as-a-short-term-free-lance-contract." class="section level6">
<h6>Worked to develop an object-oriented back-end in R for organizing, modeling, and analyzing transactions between financial institutions. Position held as a short-term, free-lance contract.</h6>
</div>
</div>
</div>
<div id="in-research" class="section level4">
<h4>In research,</h4>
<div id="suny-upstate-medical-university-august-2012-to-may-2016" class="section level5">
<h5><strong>SUNY Upstate Medical University:</strong> August 2012 to May 2016</h5>
<div id="worked-in-a-psychiatric-genetics-lab-under-dr.stephen-glatt-to-study-the-molecular-underpinnings-of-psychiatric-disorders.-efforts-largely-included-the-development-of-bioinformatic-tools-for-the-analysis-of-single-nucleotide-polymorphisms-and-microarray-based-gene-expression-data.-extensive-time-devoted-to-the-implementation-machine-learning-for-the-classification-of-autism-spectrum-disorder-using-genomic-data." class="section level6">
<h6>Worked in a psychiatric genetics lab under Dr. Stephen Glatt to study the molecular underpinnings of psychiatric disorders. Efforts largely included the development of bioinformatic tools for the analysis of single nucleotide polymorphisms and microarray-based gene expression data. Extensive time devoted to the implementation machine learning for the classification of autism spectrum disorder using genomic data.</h6>
</div>
</div>
<div id="suny-new-paltz-september-2009-to-august-2010" class="section level5">
<h5><strong>SUNY New Paltz:</strong> September 2009 to August 2010</h5>
<div id="worked-in-an-organic-chemistry-lab-under-dr.frantz-folmer-andersen-to-study-synthetic-diamine-macrocycles-which-have-the-capability-to-distinguish-between-mirror-image-molecules." class="section level6">
<h6>Worked in an organic chemistry lab under Dr. Frantz Folmer-Andersen to study synthetic diamine macrocycles which have the capability to distinguish between mirror image molecules.</h6>
</div>
</div>
<div id="suny-purchase-may-2009-to-july-2009" class="section level5">
<h5><strong>SUNY Purchase:</strong> May 2009 to July 2009</h5>
<div id="worked-in-an-environmental-science-lab-under-oceanographer-dr.kristopher-baker-to-study-the-impact-of-anthropogenic-input-on-the-biodiversity-of-the-hudson-river-estuary.-investigation-presented-as-lecture." class="section level6">
<h6>Worked in an environmental science lab under oceanographer Dr. Kristopher Baker to study the impact of anthropogenic input on the biodiversity of the Hudson River estuary. Investigation presented as lecture.</h6>
</div>
</div>
</div>
<div id="in-education" class="section level4">
<h4>In education,</h4>
<div id="deakin-university-march-2017-to-june-2017" class="section level5">
<h5><strong>Deakin University:</strong> March 2017 to June 2017</h5>
<div id="employed-as-a-graduate-assistant-for-an-undergraduate-course-titled-principles-of-pharmacology." class="section level6">
<h6>Employed as a graduate assistant for an undergraduate course titled, <em>Principles of Pharmacology</em>.</h6>
</div>
</div>
<div id="coursera-september-2016-to-october-2016" class="section level5">
<h5><strong>Coursera:</strong> September 2016 to October 2016</h5>
<div id="employed-as-a-graduate-assistant-for-an-online-course-titled-big-data-genes-and-medicine." class="section level6">
<h6>Employed as a graduate assistant for an online course titled, <em>Big Data, Genes, and Medicine</em>.</h6>
</div>
</div>
<div id="suny-new-paltz-september-2009-to-may-2011" class="section level5">
<h5><strong>SUNY New Paltz:</strong> September 2009 to May 2011</h5>
<div id="employed-as-a-teaching-assistant-for-an-undergraduate-organic-chemistry-laboratory-class.-periodically-employed-as-a-tutor-to-undergraduate-students-in-the-subjects-of-mathematics-biology-and-chemistry." class="section level6">
<h6>Employed as a teaching assistant for an undergraduate organic chemistry laboratory class. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.</h6>
</div>
</div>
<div id="suny-orange-september-2008-to-august-2009" class="section level5">
<h5><strong>SUNY Orange:</strong> September 2008 to August 2009</h5>
<div id="employed-in-a-math-learning-center-to-assist-students-in-need-of-help-in-college-mathematics-courses.-periodically-employed-as-a-tutor-to-undergraduate-students-in-the-subjects-of-mathematics-biology-and-chemistry." class="section level6">
<h6>Employed in a math learning center to assist students in need of help in college mathematics courses. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.</h6>
<p><br></p>
</div>
</div>
</div>
</div>
<div id="awards-and-scholarships" class="section level3">
<h3>Awards and scholarships</h3>
<p>2016 CSL Travel Award to 2nd Bioconductor Asia-Pacific Developers’ Meeting</p>
<p>2014 Student Research Celebration Presenter Research Project Award</p>
<p>2010 New Paltz Student Undergraduate Research Experience (SURE) stipend</p>
<p>2009 Purchase Bridges to Baccalaureate Program stipend</p>
<p>2009 Outgoing Faculty Annual Scholarship awarded by SUNY Orange</p>
<p>2009 Outgoing William F. Ehlers Jr. Memorial Scholarship for excellence in anthropology awarded by SUNY Orange</p>
<p>2009 Outgoing Dr. Cortland R. Mapes Scholarship for excellence in biology awarded by SUNY Orange</p>
<p>2008 American Chemical Society award for excellence in chemistry</p>
<p>2007-2009 Recipient of President’s Scholarship for Outstanding Academic Achievement at SUNY Orange</p>
<p><br></p>
</div>
<div id="publications" class="section level3">
<h3>Publications</h3>
<p><strong>Quinn T</strong>, Richardson MF, Lovell D, &amp; Crowley T. 2017. propr: An R-package for Identifying Proportionally Abundant Features Using Compositional Data Analysis. bioRxiv 104935. <a href="http://dx.doi.org/10.1101/104935" class="uri">http://dx.doi.org/10.1101/104935</a>.</p>
<p>Tylee DS, Hess JL, <strong>Quinn TP</strong>, et al., &amp; Glatt SJ. 2016. Blood transcriptomic comparison of individuals with and without autism spectrum disorder: A combined-samples mega-analysis. American Journal of Medical Genetics B Neuropsychiatric Genetics. Rapid Publication. <a href="http://dx.doi.org/10.1002/ajmg.b.32511" class="uri">http://dx.doi.org/10.1002/ajmg.b.32511</a>.</p>
<p><strong>Quinn T</strong>, Tylee D, &amp; Glatt S. 2016. exprso: an R-package for the rapid implementation of machine learning algorithms. F1000Research 5:2588. <a href="http://dx.doi.org/10.12688/f1000research.9893.1" class="uri">http://dx.doi.org/10.12688/f1000research.9893.1</a>.</p>
<p>Bichindaritz IB, Cole E, Keshan N, Parimi P, &amp; <strong>Quinn TP</strong>. 2016. Machine Learning Based Automatic Multilevel Stress Detection from ECG Signals. IJCAI 2016: Workshop on Knowledge Discovery in Healthcare Data.</p>
<p>Hess JL, <strong>Quinn TP</strong>, Akbarian S, &amp; Glatt SJ. 2015. Bioinformatic analyses and conceptual synthesis of evidence linking ZNF804A to risk for schizophrenia and bipolar disorder. American Journal of Medical Genetics B Neuropsychiatric Genetics 168(1):14-35. <a href="http://dx.doi.org/10.1002/ajmg.b.32284" class="uri">http://dx.doi.org/10.1002/ajmg.b.32284</a>.</p>
<p><strong>Quinn TP</strong>, Atwood PD, Tanski JM, Moore TF, &amp; Folmer-Andersen J. 2011. Aza-crown macrocycles as chiral solvating agents for mandelic acid derivatives. Journal of Organic Chemistry 76(24):10020-30. <a href="http://dx.doi.org/10.1021/jo2018203" class="uri">http://dx.doi.org/10.1021/jo2018203</a>.</p>
<p><br></p>
</div>
</div>

<div style="position: relative">
  <p style="position: relative; bottom: 0; width:100%; text-align: center; font-size: small">
    &copy; 2017 | <a href="mailto:thom@tpq.me">thom@tpq.me</a> |
    <a href="http://twitter.com/tpq__">Twitter</a> |
    <a href="http://github.com/tpq"> GitHub</a>
  </p>
</div>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
