<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Curriculum Vitae</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<link rel="icon" type="image/x-icon" href="favicon.ico" />




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">THOM QUINN</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="about-me.html">ABOUT ME</a>
</li>
<li>
  <a href="curriculum-vitae.html">CURRICULUM VITAE</a>
</li>
<li>
  <a href="musings.html">MUSINGS</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Curriculum Vitae</h1>

</div>


<div id="education" class="section level3">
<h3>Education</h3>
</div>
<div id="phd" class="section level2">
<h2>PhD</h2>
<div id="deakin-university-september-2016-to-present" class="section level6">
<h6><strong>Deakin University:</strong> September 2016 to present</h6>
</div>
<div id="currently-pursuing-a-doctor-of-philosophy-with-a-concentration-in-bioinformatics-and-medical-genomics" class="section level6">
<h6>Currently pursuing a Doctor of Philosophy with a concentration in bioinformatics and medical genomics</h6>
</div>
</div>
<div id="md" class="section level2">
<h2>MD</h2>
<div id="suny-upstate-medical-university-august-2011-to-may-2016" class="section level6">
<h6><strong>SUNY Upstate Medical University:</strong> August 2011 to May 2016</h6>
</div>
<div id="graduated-with-a-medicinae-doctor-devoting-credited-hours-to-research-in-bioinformatics-and-medical-genomics" class="section level6">
<h6>Graduated with a Medicinae Doctor, devoting credited hours to research in bioinformatics and medical genomics</h6>
</div>
</div>
<div id="bs" class="section level2">
<h2>BS</h2>
<div id="suny-new-paltz-august-2009-to-may-2011" class="section level6">
<h6><strong>SUNY New Paltz:</strong> August 2009 to May 2011</h6>
</div>
<div id="graduated-summa-cum-laude-obtaining-a-bachelors-degree-in-biology-with-a-concentration-in-biotechnology" class="section level6">
<h6>Graduated <em>summa cum laude</em>, obtaining a Bachelor’s Degree in Biology with a concentration in biotechnology</h6>
</div>
</div>
<div id="as" class="section level2">
<h2>AS</h2>
<div id="suny-orange-january-2007-to-may-2009" class="section level6">
<h6><strong>SUNY Orange:</strong> January 2007 to May 2009</h6>
</div>
<div id="graduated-summa-cum-laude-obtaining-an-associates-degree-in-science-with-a-concentration-in-biology" class="section level6">
<h6>Graduated <em>summa cum laude</em>, obtaining an Associate’s Degree in Science with a concentration in biology</h6>
<p><br></p>
</div>
<div id="professional-skills" class="section level3">
<h3>Professional skills</h3>
<div id="rbioconductor" class="section level5">
<h5><strong>R/Bioconductor</strong></h5>
<div id="fluency-in-the-r-scripting-language-with-experience-using-several-bioconductor-packages.-some-bioinformatic-programming-projects-include-the-development-of-tools-for-predicting-transcription-factor-binding-motif-disruptions-by-disorder-associated-snps-mapping-molecular-interactions-using-multiple-databases-and-implementing-high-throughput-supervised-machine-learning-with-sample-bootstrapping." class="section level6">
<h6>Fluency in the R scripting language with experience using several Bioconductor packages. Some bioinformatic programming projects include the development of tools for predicting transcription factor binding motif disruptions by disorder-associated SNPs, mapping molecular interactions using multiple databases, and implementing high-throughput supervised machine learning with sample bootstrapping.</h6>
</div>
</div>
<div id="bioinformatics" class="section level5">
<h5><strong>Bioinformatics</strong></h5>
<div id="familiarity-using-public-data-repositories-including-dbgap-ncbi-geo-biomart-and-ucsc-genome-browser.-extensive-time-devoted-to-applying-supervised-machine-learning-in-the-classification-of-patients-based-on-gene-expression-data-including-the-exploration-of-novel-feature-selection-methods.-familiarity-with-clustering-algorithms-including-hierarchical-clustering-and-k-means-clustering.-experience-visualizing-high-dimensional-data-through-dimension-reduction-with-methods-such-as-principal-components-analysis." class="section level6">
<h6>Familiarity using public data repositories including dbGAP, NCBI GEO, BioMart, and UCSC Genome Browser. Extensive time devoted to applying supervised machine learning in the classification of patients based on gene expression data, including the exploration of novel feature selection methods. Familiarity with clustering algorithms including hierarchical clustering and k-means clustering. Experience visualizing high-dimensional data through dimension reduction with methods such as principal components analysis.</h6>
<p><br></p>
</div>
</div>
</div>
<div id="professional-experience" class="section level3">
<h3>Professional experience</h3>
<div id="in-industry" class="section level4">
<h4>In industry,</h4>
<div id="adveq-management-april-2016-to-present" class="section level5">
<h5><strong>Adveq Management:</strong> April 2016 to present</h5>
<div id="worked-to-develop-an-object-oriented-back-end-in-r-for-organizing-modeling-and-analyzing-transactions-between-financial-institutions.-position-held-as-a-short-term-free-lance-contract." class="section level6">
<h6>Worked to develop an object-oriented back-end in R for organizing, modeling, and analyzing transactions between financial institutions. Position held as a short-term, free-lance contract.</h6>
</div>
</div>
</div>
<div id="in-research" class="section level4">
<h4>In research,</h4>
<div id="suny-upstate-medical-university-august-2012-to-may-2016" class="section level5">
<h5><strong>SUNY Upstate Medical University:</strong> August 2012 to May 2016</h5>
<div id="worked-in-a-psychiatric-genetics-lab-under-dr.stephen-glatt-to-study-the-molecular-underpinnings-of-psychiatric-disorders.-efforts-largely-included-the-development-of-bioinformatic-tools-for-the-analysis-of-single-nucleotide-polymorphisms-and-microarray-based-gene-expression-data.-extensive-time-devoted-to-the-implementation-machine-learning-for-the-classification-of-autism-spectrum-disorder-using-genomic-data." class="section level6">
<h6>Worked in a psychiatric genetics lab under Dr. Stephen Glatt to study the molecular underpinnings of psychiatric disorders. Efforts largely included the development of bioinformatic tools for the analysis of single nucleotide polymorphisms and microarray-based gene expression data. Extensive time devoted to the implementation machine learning for the classification of autism spectrum disorder using genomic data.</h6>
</div>
</div>
<div id="suny-new-paltz-september-2009-to-august-2010" class="section level5">
<h5><strong>SUNY New Paltz:</strong> September 2009 to August 2010</h5>
<div id="worked-in-an-organic-chemistry-lab-under-dr.frantz-folmer-andersen-to-study-synthetic-diamine-macrocycles-which-have-the-capability-to-distinguish-between-mirror-image-molecules." class="section level6">
<h6>Worked in an organic chemistry lab under Dr. Frantz Folmer-Andersen to study synthetic diamine macrocycles which have the capability to distinguish between mirror image molecules.</h6>
</div>
</div>
<div id="suny-purchase-may-2009-to-july-2009" class="section level5">
<h5><strong>SUNY Purchase:</strong> May 2009 to July 2009</h5>
<div id="worked-in-an-environmental-science-lab-under-oceanographer-dr.kristopher-baker-to-study-the-impact-of-anthropogenic-input-on-the-biodiversity-of-the-hudson-river-estuary.-investigation-presented-as-lecture." class="section level6">
<h6>Worked in an environmental science lab under oceanographer Dr. Kristopher Baker to study the impact of anthropogenic input on the biodiversity of the Hudson River estuary. Investigation presented as lecture.</h6>
</div>
</div>
</div>
<div id="in-education" class="section level4">
<h4>In education,</h4>
<div id="coursera-september-2016-to-october-2016" class="section level5">
<h5><strong>Coursera:</strong> September 2016 to October 2016</h5>
<div id="employed-as-a-graduate-assistant-to-dr.isabelle-bichindaritz-for-an-online-course-titled-big-data-genes-and-medicine." class="section level6">
<h6>Employed as a graduate assistant to Dr. Isabelle Bichindaritz for an online course titled <em>Big Data, Genes, and Medicine</em>.</h6>
</div>
</div>
<div id="suny-new-paltz-september-2009-to-may-2011" class="section level5">
<h5><strong>SUNY New Paltz:</strong> September 2009 to May 2011</h5>
<div id="employed-as-a-teaching-assistant-for-an-undergraduate-organic-chemistry-laboratory-class.-periodically-employed-as-a-tutor-to-undergraduate-students-in-the-subjects-of-mathematics-biology-and-chemistry." class="section level6">
<h6>Employed as a teaching assistant for an undergraduate organic chemistry laboratory class. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.</h6>
</div>
</div>
<div id="suny-orange-september-2008-to-august-2009" class="section level5">
<h5><strong>SUNY Orange:</strong> September 2008 to August 2009</h5>
<div id="employed-in-a-math-learning-center-to-assist-students-in-need-of-help-in-college-mathematics-courses.-periodically-employed-as-a-tutor-to-undergraduate-students-in-the-subjects-of-mathematics-biology-and-chemistry." class="section level6">
<h6>Employed in a math learning center to assist students in need of help in college mathematics courses. Periodically employed as a tutor to undergraduate students in the subjects of mathematics, biology, and chemistry.</h6>
<p><br></p>
</div>
</div>
</div>
</div>
<div id="awards-and-scholarships" class="section level3">
<h3>Awards and scholarships</h3>
<p>2016 CSL Travel Award to 2nd Bioconductor Asia-Pacific Developers’ Meeting</p>
<p>2014 Student Research Celebration Presenter Research Project Award</p>
<p>2010 New Paltz Student Undergraduate Research Experience (SURE) stipend</p>
<p>2009 Purchase Bridges to Baccalaureate Program stipend</p>
<p>2009 Outgoing Faculty Annual Scholarship awarded by SUNY Orange</p>
<p>2009 Outgoing William F. Ehlers Jr. Memorial Scholarship for excellence in anthropology awarded by SUNY Orange</p>
<p>2009 Outgoing Dr. Cortland R. Mapes Scholarship for excellence in biology awarded by SUNY Orange</p>
<p>2008 American Chemical Society award for excellence in chemistry</p>
<p>2007-2009 Recipient of President’s Scholarship for Outstanding Academic Achievement at SUNY Orange</p>
<p><br></p>
</div>
<div id="presentations" class="section level3">
<h3>Presentations</h3>
<p><strong>Quinn T</strong>, Richardson M, Lovell D, &amp; Crowley T. October 2016. propr: An R Package for Identifying Proportionally Expressed Genes using Compositional Data Analysis. Poster presented at Australasian Genomic Technologies Association in Auckland, New Zealand.</p>
<p><strong>Quinn TP</strong>, Tylee DS, Bichindaritz IB, &amp; Glatt SJ. April 2016. Constructing Generalizable Classifiers for the Classification of Complex Biological Disorders. Poster presented at SUNY Upstate Medical University Student Research Day in Syracuse, New York.</p>
<p>Tylee DS, Hess JL, <strong>Quinn TP</strong>, Barve R, &amp; Glatt SJ. April 2016. Transcriptome-wide mega-analysis of blood-based microarray data comparing individuals with autism and typically developing subjects. Poster presented at SUNY Upstate Medical University Student Research Day in Syracuse, New York.</p>
<p><strong>Quinn TP</strong>, Tylee DS, Bichindaritz IB, &amp; Glatt SJ. March 2016. Constructing Generalizable Classifiers for the Classification of Complex Biological Disorders. Poster presented at The NYAS Annual Machine Learning Symposium in New York, New York.</p>
<p>Tylee DS, Hess JL, <strong>Quinn TP</strong>, Barve R, &amp; Glatt SJ. October 2015. Transcriptome-wide mega-analysis of blood-based microarray data comparing individuals with autism and typically developing subjects. Poster presented at World Congress of Psychiatric Genetics in Toronto, Ontario.</p>
<p>Tylee DS, <strong>Quinn TP</strong>, &amp; Glatt SJ. June 2015. Improving SVM classifiers using resampling and aggregation: A blood-based transcriptomic study of children with autism. Poster presented at Cold Spring Harbor Workshop on Autism Spectrum Disorders in Cold Spring Harbor, New York.</p>
<p>Tylee DS, <strong>Quinn TP</strong>, &amp; Glatt SJ. October 2014. Improving SVM classifiers using resampling and aggregation: A blood-based transcriptomic study of children with autism. Poster presented at World Congress of Psychiatric Genetics in Copenhagen, Denmark.</p>
<p>Glatt SJ, <strong>Quinn TP</strong>, &amp; Tylee DS. October 2014. Building better classifiers: optimizing models of heterogeneous neuropsychiatric disorders derived from high-dimensional data. Poster presented at World Congress of Psychiatric Genetics in Copenhagen, Denmark.</p>
<p><strong>Quinn TP</strong>, Hess JL, &amp; Glatt SJ. April 2014. In silico discovery of transcription factor-binding motifs disrupted by single nucleotide polymorphisms associated with multiple psychiatric disorders. Poster presented at SUNY Upstate Medical University Student Research Day in Syracuse, New York.</p>
<p><strong>Quinn TP</strong>, Minicozzi M, &amp; Mass S. April 2011. Biometrics of metamorphosed and neotenic axolotls. Poster presented at SUNY New Paltz Student Research Symposium in New Paltz, New York.</p>
<p><strong>Quinn TP</strong>. September 2010. New chiral shift reagents. Lecture presented through the SUNY New Paltz Student Undergraduate Research Experience (SURE) Program in New Paltz, New York.</p>
<p><strong>Quinn TP</strong> &amp; Folmer-Andersen J. April 2010. New macrocycles as chiral shift reagents. Poster presented at SUNY New Paltz Student Research Symposium in New Paltz, New York.</p>
<p><strong>Quinn TP</strong> &amp; Folmer-Andersen J. April 2010. New macrocycles as chiral shift reagents. Poster presented at Bard University in Red Hook, New York.</p>
<p><strong>Quinn TP</strong>. July 2009. Anthropogenic inputs in the Hudson River estuary. Lecture presented through the SUNY Purchase Bridges to Baccalaureate Program in Harrison, New York.</p>
<p><strong>Quinn TP</strong>. July 2009. The Role of Horizontal and Vertical Learning in Chimpanzee Populations. Lecture presented at Beacon Conference for Student Scholars at Two-Year Colleges in Takoma Park, MD.</p>
<p><br></p>
</div>
<div id="publications" class="section level3">
<h3>Publications</h3>
<p><strong>Quinn T</strong>, Tylee D, &amp; Glatt S. 2016. exprso: an R-package for the rapid implementation of machine learning algorithms. F1000Research 5:2588.</p>
<p>Bichindaritz IB, Cole E, Keshan N, Parimi P, &amp; <strong>Quinn TP</strong>. 2016. Machine Learning Based Automatic Multilevel Stress Detection from ECG Signals. IJCAI 2016: Workshop on Knowledge Discovery in Healthcare Data.</p>
<p>Hess JL, <strong>Quinn TP</strong>, Akbarian S, &amp; Glatt SJ. 2015. Bioinformatic analyses and conceptual synthesis of evidence linking ZNF804A to risk for schizophrenia and bipolar disorder. American Journal of Medical Genetics B Neuropsychiatric Genetics 168(1):14-35.</p>
<p><strong>Quinn TP</strong>, Atwood PD, Tanski JM, Moore TF, &amp; Folmer-Andersen J. 2011. Aza-crown macrocycles as chiral solvating agents for mandelic acid derivatives. Journal of Organic Chemistry 76(24):10020-30.</p>
<p><br></p>
</div>
</div>

<div style="position: relative">
  <p style="position: relative; bottom: 0; width:100%; text-align: center; font-size: small">
    &copy; 2016 | <a href="mailto:thom@tpq.me">thom@tpq.me</a> |
    <a href="http://twitter.com/tpq__">Twitter</a> |
    <a href="http://github.com/tpq"> GitHub</a>
  </p>
</div>



</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
